Source/Core/DolphinQt/Config/Mapping/IOWindow.cpp

```diff
@@ -145,17 +162,38 @@ void IOWindow::ConnectWidgets()
           this, &IOWindow::OnRangeChanged);
   connect(m_range_slider, static_cast<void (QSlider::*)(int value)>(&QSlider::valueChanged), this,
           &IOWindow::OnRangeChanged);
+
+  connect(m_expression_text, &QPlainTextEdit::textChanged, [this] {
+    m_apply_button->setText(m_apply_button->text().remove(QStringLiteral("*")));
+    m_apply_button->setText(m_apply_button->text() + QStringLiteral("*"));
+  });
+
+  connect(m_operators_combo, QOverload<int>::of(&QComboBox::activated), [this](int index) {
+    if (0 == index)
```

Comment:
Are we certain that the separator item doesn't count for those indices? Also goes for the function combo down below.

Reply:
The separator is after the first item so it doesn't matter as I only use the index for checking that the first item (the label) hasn't been selected.

Reply:
Well yeah, but the separator will probably blow up on `currentText().left(1)` (if it even has a text; and not just blow up in general for both calls)

Reply:
The separator is not selectable. It's just a visual separator.

Reply:
That also works 👍 
Just making sure things don't explode.

Reply:
:+1: